rpcs3/Emu/RSX/VK/vkutils/swapchain.hpp

```diff
@@ -586,8 +588,49 @@ namespace vk
 			VkSwapchainKHR old_swapchain = m_vk_swapchain;
 			vk::physical_device& gpu = const_cast<vk::physical_device&>(dev.gpu());
 
-			VkSurfaceCapabilitiesKHR surface_descriptors = {};
-			CHECK_RESULT(vkGetPhysicalDeviceSurfaceCapabilitiesKHR(gpu, m_surface, &surface_descriptors));
+			VkSurfaceCapabilities2KHR pSurfaceCapabilities = {};
+			pSurfaceCapabilities.sType                     = VK_STRUCTURE_TYPE_SURFACE_CAPABILITIES_2_KHR;
+
+			VkPhysicalDeviceSurfaceInfo2KHR pSurfaceInfo = {};
+			pSurfaceInfo.sType                           = VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_SURFACE_INFO_2_KHR;
+			pSurfaceInfo.surface                         = m_surface;
+
+			VkSurfaceCapabilitiesKHR surface_descriptors;
+			bool init_surface_descriptors = true;
+			bool should_disable_exclusive_full_screen = false;

```

Comment:
I think this can also be moved into the windows section

Reply:
`should_disable_exclusive_full_screen` yes, but not the `surface_descriptors` or the flag that it's still uninitialized

Reply:
Which is why I only marked this line 😹